src/mol-gl/webgl/uniform.ts

```diff
@@ -42,6 +42,14 @@ export function getUniformType(gl: GLRenderingContext, kind: UniformKind) {
     }
 }
 
+export function isArrayUniform(kind: UniformKind) {
```

Comment:
I think just `return kind.endsWith('[]');` is fine

Reply:
Sure, simplified with [cbc941f](https://github.com/molstar/molstar/pull/434/commits/cbc941f19388c222f501ebf5520c5a36adcedd6f).

Changelog is updated as well.